### DISCUSSION TITLE
新人正式入网（黄泉老婆美图镇楼）

### DISCUSSION THREAD

**Discussion by HaiChooseWang** [share121/inter-knot#452] [其他]
大家好，这里是海折惘，是的HaiChooseWang的意思就是这个哦。在B站上刷到了up主的视频，那身为舰长、开拓者、绳匠的我怎么也得来帮帮场子。

![image](https://github.com/user-attachments/assets/2de59f23-cfe3-4253-8f09-fefc206320dd)

**Comment by wudijians123** [comment]
支持
![Uploading F9Y1W9)7_AU)%RFT~~TE`TT.jpg…]()
